mmv1/third_party/terraform/website/docs/d/vmwareengine_nsx_credentials.html.markdown

```diff
@@ -0,0 +1,33 @@
+---
+subcategory: "Cloud VMware Engine"
+description: |-
+  Get NSX Credentials of a Private Cloud.
+---
+
+# google\_vmwareengine\_nsx_credentials
+
+Use this data source to get NSX credentials for a Private Cloud.
+
+To get more information about private cloud NSX credentials, see:
+* [API documentation](https://cloud.google.com/vmware-engine/docs/reference/rest/v1/projects.locations.privateClouds/showNsxCredentials)
+
+## Example Usage
+
+```hcl
+data "google_vmwareengine_nsx_credentials" "ds" {
+	parent =  "project/locations/us-west1-a/privateClouds/my-cloud"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `parent` - (Required) The resource name of the private cloud which contains the NSX.
```

Comment:
The nsx credentials resource is a child of private cloud and can't be used without a private cloud configured. Hence, I wanted to preserve the same relation here as well.

Reply:
I'm worried that it might make things more complicated for the user. If we already have the location input for the parent private cloud, do we need to have it separately here?

Reply:
It's fairly standard to have the user input location and project again for a child resource, but we can keep it as parent.